spaceinvaders/src/screen-logic/screens/PlayScreen.cs

```diff
@@ -185,6 +180,10 @@ private void EnemiesUpdate()
         _redEnemy?.Update();
     }
 
+    private void EnemiesLogicUpdate() {
```

Comment:
Se for possível colocar todos os **updates** (se for possível) dentro de alguma lista e fazer o update de cada um que foi inserido na lista de updatable, se não vai ficar um negócio **gigante**, pois quando terminar um novo componente do jogo, já adiciona nessa lista de updatables.

Reply:
Vou fazer isso em outra branch, vou mover toda a lógica de todos os updates em uma outra classe diferente.

Reply:
Joia